### PR TITLE
added ALLOW_SECURE_COOKIES env variable & made minor changes to docs

### PR DESCRIPTION
**Pull request by SanskritiHarmukh**
- [x] Updated the environment file in Community and Enterprise Edition with new variable : ALLOW_SECURE_COOKIES
- [x] Changed "My workspace" to "Personal Workspace" in [Workspaces page](https://docs.hoppscotch.io/documentation/features/workspaces).
- [x] Edited the description for [Buttons widget](https://docs.hoppscotch.io/documentation/features/widgets#buttons), since only 2 options are available to copy the format from- HTML and Markdown.

### CODE REVIEW

File: documentation/self-host/enterprise-edition/install-and-build.mdx [hunk 1]
```diff
@@ -136,25 +140,26 @@ Let's understand the major environment variables:
...
-18. `MAILER_SMTP_PASSWORD`: Provide the password set for the SMTP user.
-19. `RATE_LIMIT_TTL`: The time it takes to refresh the maximum number of requests being received.
-20. `RATE_LIMIT_MAX`: The maximum number of requests that Hoppscotch can handle under `RATE_LIMIT_TTL`.
-21. `VITE_BASE_URL`: This is the URL where your deployment will be accessible from.
-22. `VITE_SHORTCODE_BASE_URL`: A URL to generate shortcodes for sharing, can be the same as `VITE_BASE_URL`.
-23. `VITE_BACKEND_GQL_URL`: The URL for GraphQL within the instance.
-24. `VITE_BACKEND_WS_URL`: The URL for WebSockets within the instance.
-25. `VITE_BACKEND_API_URL`: The URL for REST APIs within the instance.
-26. `VITE_APP_TOS_LINK` and `VITE_APP_PRIVACY_POLICY_LINK` are optional and are used to configure the links to the Terms & Conditions and Privacy Policy.
+8. `ALLOW_SECURE_COOKIES` : Ensures that cookies are transmitted only over secure (HTTPS) connections.
```
Comment (balub) [inline documentation/self-host/enterprise-edition/install-and-build.mdx:143]:
```suggestion
8. `ALLOW_SECURE_COOKIES`: If disabled users will be able to use Hoppscotch over HTTP connections as well. It is recommended that this be left enabled as some auth providers may not work if this value is set to true.
```

File: documentation/self-host/community-edition/install-and-build.mdx [hunk 2]
```diff
@@ -94,25 +98,26 @@ Let's understand the major environment variables:
...
-17. `MAILER_SMTP_PASSWORD`: Provide the password set for the SMTP user.
-18. `RATE_LIMIT_TTL`: The time it takes to refresh the maximum number of requests being received.
-19. `RATE_LIMIT_MAX`: The maximum number of requests that Hoppscotch can handle under `RATE_LIMIT_TTL`.
-20. `VITE_BASE_URL`: This is the URL where your deployment will be accessible from.
-21. `VITE_SHORTCODE_BASE_URL`: A URL to generate shortcodes for sharing, can be the same as `VITE_BASE_URL`.
-22. `VITE_BACKEND_GQL_URL`: The URL for GraphQL within the instance.
-23. `VITE_BACKEND_WS_URL`: The URL for WebSockets within the instance.
-24. `VITE_BACKEND_API_URL`: The URL for REST APIs within the instance.
-25. `VITE_APP_TOS_LINK` and `VITE_APP_PRIVACY_POLICY_LINK` are optional and are used to configure the links to the Terms & Conditions and Privacy Policy.
+7. `ALLOW_SECURE_COOKIES` : Ensures that cookies are transmitted only over secure (HTTPS) connections.
```
Comment (balub) [inline documentation/self-host/community-edition/install-and-build.mdx:101]:
```suggestion
7. `ALLOW_SECURE_COOKIES`: If disabled users will be able to use Hoppscotch over HTTP connections as well. It is recommended that this be left enabled as some auth providers may not work if this value is set to true.
```

File: documentation/features/widgets.mdx
Comment (balub) [inline documentation/features/widgets.mdx:1]: I don't see how this change is relevant to the PR, please remove it from this PR.

**Review by balub — CHANGES_REQUESTED** [review]

File: documentation/features/widgets.mdx
Comment (SanskritiHarmukh) [reply documentation/features/widgets.mdx:1]: Okay, I spotted it while going through docs, thus made changes in this PR itself.

Comment (balub) [reply documentation/features/widgets.mdx:1]: I would suggest mentioning this in the PR desc or someplace else. I have no context regarding this change hence don't want to approve and have something possible wrong merged.

Comment (SanskritiHarmukh) [reply documentation/features/widgets.mdx:1]: Sure, I have updated the description. Will ask Kiran to review.

**Review by thetronjohnson — APPROVED** [review]

**Pull request merged**